protocol.html

```diff
@@ -1301,270 +1435,445 @@ <h3 property="schema:name">Internationalization Considerations</h3>
+                  <tr about="#b3536d3a-0d3a-40c9-997b-07aa2f65b5d3" typeof="spec:Change">
+                    <td><a href="https://www.w3.org/Consortium/Process/#class-1" rel="spec:changeClass" resource="spec:ChangeClass1">1</a></td>
+                    <td><a href="" rel="spec:changeSubject">Document</a></td>
+                    <td property="spec:statement">Amend broken links, style sheets, or invalid markup.</td>
+                  </tr>
+                  <tr about="#fa50e3ae-ca0d-4857-877c-b88c18dc21ce" typeof="spec:Change">
+                    <td><a href="https://www.w3.org/Consortium/Process/#class-2" rel="spec:changeClass" resource="spec:ChangeClass2">2</a></td>
+                    <td><a href="" rel="spec:changeSubject">Document</a></td>
+                    <td property="spec:statement">Amend language and document details.</td>
+                  </tr>
+                  <tr about="#d18392b9-8ffc-4264-a5a2-10e2da6fac32" typeof="spec:Change">
+                    <td><a href="https://www.w3.org/Consortium/Process/#class-4" rel="spec:changeClass" resource="spec:ChangeClass4">4</a></td>
+                    <td><a href="#server-content-type-includes" rel="spec:changeSubject">#server-content-type-includes</a></td>
+                    <td property="spec:statement">Add requirement for server to include <code>Content-Type</code> in <a href="#server-content-type-includes">messages with content</a>.</td>
+                  </tr>
+                  <tr about="#a91960b6-0543-4acc-8153-689c435c7659" typeof="spec:Change">
+                    <td><a href="https://www.w3.org/Consortium/Process/#class-2" rel="spec:changeClass" resource="spec:ChangeClass2">2</a></td>
+                    <td><a href="#relationship-to-ldp" rel="spec:changeSubject">#relationship-to-ldp</a></td>
+                    <td property="spec:statement">Add section describing the <a href="#relationship-to-ldp">the relationship between Solid Protocol and LDP</a>.</td>
+                  </tr>
+                  <tr about="#b0af4fa6-5d35-47b5-8fe3-ad597dc8bdb5" typeof="spec:Change">
+                    <td><a href="https://www.w3.org/Consortium/Process/#class-2" rel="spec:changeClass" resource="spec:ChangeClass2">2</a></td>
+                    <td><a href="#server-representation-turtle-jsonld" rel="spec:changeSubject">#server-representation-turtle-jsonld</a></td>
+                    <td property="spec:statement">Amend the server requirement emphasising <a href="#server-representation-turtle-jsonld">creating RDF sources</a> and their representations.</td>
+                  </tr>
+                  <tr about="#afd2402a-3174-4f27-af9d-f80461538e95" typeof="spec:Change">
+                    <td><a href="https://www.w3.org/Consortium/Process/#class-2" rel="spec:changeClass" resource="spec:ChangeClass2">2</a></td>
+                    <td><a href="#server-put-patch-uri-assignment" rel="spec:changeSubject">#server-put-patch-uri-assignment</a></td>
```

Comment:
The anchor pointing to `#server-put-patch-uri-assignment` is broken. It is talking about something being removed, so I guess that means it doesn't exist anymore. But in that case, shouldn't it link to an old version of the spec or something?

Reply:
Great catch, thanks! I totally forgot to come back to this. Fixed in ED: https://github.com/solid/specification/commit/4aa056b0617fbb58ea078a970df7b27ca0388893 . As mentioned above, I'll update this PR after other editorial changes.